nautobot_netbox_importer/tests/test_import_objectchange.py

```diff
@@ -0,0 +1,121 @@
+"""Test the importing functionality of nautobot-netbox-importer."""
+
+import os
+import json
+import copy
+from unittest.mock import Mock
+
+import yaml
+from django.core.management import call_command
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+from nautobot.extras.models import ObjectChange
+from nautobot.extras import models
+from nautobot_netbox_importer.management.commands.import_netbox_objectchange_json import Command
+from .test_import import TestImport
+
+
+NETBOX_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_dump.json")
+NETBOX_OBJECTCHANGE_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "netbox_objectchange_dump.json")
+NAUTOBOT_DATA_FILE = os.path.join(os.path.dirname(__file__), "fixtures", "nautobot_expectations.yaml")
+NAUTOBOT_OBJECTCHANGE_DATA_FILE = os.path.join(
+    os.path.dirname(__file__), "fixtures", "nautobot_objectchange_expectations.yaml"
+)
+
+
+class TestImportObjectChange(TestImport):
+    """Test the importing functionality of nautobot-netbox-importer, for ObjectChange objects.
+
+    It is mostly reusing same tests as TestImport, but with an extra command during TestData Setup.
+    """
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        """One-time setup function called before running the test functions in this class."""
+        # First we import the data without ObjectChanges
+        call_command("import_netbox_json", NETBOX_DATA_FILE, "2.10.4", verbosity=0)
+        call_command(
+            "import_netbox_objectchange_json", NETBOX_DATA_FILE, NETBOX_OBJECTCHANGE_DATA_FILE, "2.10.4", verbosity=0
+        )
+
+        # TODO check stdout/stderr for errors and such
+        with open(NAUTOBOT_DATA_FILE, "r") as handle:
+            cls.nautobot_data = yaml.safe_load(handle)
+        with open(NAUTOBOT_OBJECTCHANGE_DATA_FILE, "r") as handle:
+            cls.nautobot_data += yaml.safe_load(handle)
```

Comment:
Actually, consider using inheritance from the base `TestImport` class here, something like:

```python
def setUpTestData(cls):
    super().setUpTestData()

    with open(NETBOX_DATA_FILE, "r") as file_handle_1:
        with open(NETBOX_OBJECTCHANGE_DATA_FILE, "r") as file_handle_2:
            Command().handle(
                json_file=file_handle_1,
                objectchange_json_file=file_handle_2,
                netbox_version=version.parse("2.10.4),
                verbosity=0,
            )

    with open(NAUTOBOT_OBJECTCHANGE_DATA_FILE, "r") as handle:
        cls.nautobot_data += yaml.safe_load(handle)
```